src/mscorlib/shared/System/IO/FileStream.Win32.cs

```diff
@@ -12,10 +12,14 @@ public partial class FileStream : Stream
 
         private SafeFileHandle OpenHandle(FileMode mode, FileShare share, FileOptions options)
         {
+#if PROJECTN || CORERT
+            return CreateFile2OpenHandle(mode, share, options);
```

Comment:
The CoreRT version should be unconditionally use CreateFile. It is meant to work on pre-Win8 OSes.